src/utils/data_handler.js

```diff
@@ -693,16 +693,41 @@ var addRoutes = function(resultRoutes) {
     var row = solutionList.insertRow(nb_rows);
     row.title = 'Click to center the map';
 
+    var updateRouteOpacities = function (r, highOpacity, lowOpacity) {
+      for (var k = 0; k < routes.length; k++) {
+        if (k == r) {
+          routes[k].setStyle({opacity: highOpacity});
+        } else {
+          routes[k].setStyle({opacity: lowOpacity});
+        }
+      }
+      for (var jobMarker in data.jobsMarkers) {
+        if (data.jobsMarkers.hasOwnProperty(jobMarker)) {
+          data.jobsMarkers[jobMarker].setStyle({opacity: lowOpacity});
+        }
```

Comment:
As a result of this loop, all job makers get low opacity on first call and it never changes back again, even if they're on a route with focus. Then new jobs added with mouse click get a "normal" opacity.

The simplest way to deal with this would be to remove the loop altogether so we don't mess with markers opacity at all. What do you think?